Sources/WalletConnectSign/Sign/SignClient.swift

```diff
@@ -303,22 +326,6 @@ public final class SignClient: SignClientProtocol {
             return historyService.getPendingRequests()
         }
     }
-    
-    /// Query pending proposals
-    /// - Returns: Pending proposals received from peer with `wc_sessionPropose` protocol method
-    public func getPendingProposals(topic: String? = nil) -> [(proposal: Session.Proposal, context: VerifyContext?)] {
```

Comment:
why method was removed? I see you've added publishers but how to get initial state?

Reply:
added back